src/web/screens/dashboard.tsx

```diff
@@ -34,8 +36,11 @@ function Dashboard() {
   const { LoggedInUser, loadUser } = useContext(CurrentUserContext);
   const [showMustAccept, setShowMustAccept] = useState(false);
   const navigate = useNavigate();
+  const filteredMenu = LoggedInUser?.user?.isApprover
```

Comment:
Not sure if this is the best way to do this

Reply:
I think this is OK for just one thing like this - probably the next time we add a conditional display menu item we should come up with something better.